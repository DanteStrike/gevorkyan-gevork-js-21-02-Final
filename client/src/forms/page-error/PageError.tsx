import React from 'react';
import ContentLayout, {ContentLayoutType} from '../../components/content-layout/ContentLayout';

interface IPageErrorProps {
  title: string;
  text: string;
}

function PageError({title, text}: IPageErrorProps) {
  return (
    <ContentLayout type={ContentLayoutType.CONTENT} title={title}>
      <p>{text}</p>
    </ContentLayout>
  );
}

export default PageError;
