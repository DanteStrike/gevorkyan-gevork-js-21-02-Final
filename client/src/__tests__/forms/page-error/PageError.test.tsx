import {shallow} from 'enzyme';
import React from 'react';
import PageError from '../../../forms/page-error/PageError';

describe(`Form PageError should render correctly`, () => {
  it(`snapshot default test`, () => {
    const wrapper = shallow(<PageError title="title" text="text" />);
    expect(wrapper).toMatchSnapshot();
  });
});
