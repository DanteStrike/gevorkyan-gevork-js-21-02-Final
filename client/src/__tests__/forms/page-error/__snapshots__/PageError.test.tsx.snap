// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form PageError should render correctly snapshot default test 1`] = `
<Memo(ContentLayout)
  title="title"
  type="content"
>
  <p>
    text
  </p>
</Memo(ContentLayout)>
`;
